hosting-companies.md

```diff
@@ -61,10 +61,11 @@ In alphabetical order:
 * [WPEngine.com](http://wpengine.com)
 * [XEL](https://xel.nl)
 * [Zenbox](https://zenbox.pl)
-* [Zenith Media Canada](https://zenithmedia.ca)
+* [Zenith Media Canada](https://zenithmedia.ca/wordpress-website-hosting/)
 
 The following is a list of hosting companies that use WP-CLI, whereby their customers can write and request the running of standard and custom WP-CLI commands:
 
 * [Pressidium](https://pressidium.com)
 * [Presslabs](https://www.presslabs.com)
 * [WordPress.com VIP](http://vip.wordpress.com/)
+* [Zenith Media Canada](https://zenithmedia.ca/wordpress-website-hosting/wordpress-cli-hosting-provider/)
```

Comment:
The list is an either/or proposition here in that the requirements for the second list are already included in the first one.

If WP-CLI is installed by default in the SSH environment, you can of course also run commands with it.
```suggestion
```